content/tracing/getting_further/configure_an_apdex_for_your_traces_with_datadog_apm.md

```diff
@@ -1,6 +1,8 @@
 ---
-title: How to configure an Apdex for your traces with Datadog APM ?
-kind: faq
+title: Configure an Apdex for your traces with Datadog APM.
```

Comment:
Configure Apdex Score by Service in APM

Apdex is set and computed at the service level - not trace level.

Reply:
👌